### PR TITLE
Feature/filter country

### PR DESCRIPTION
**Pull request by tomkingston**
## Description of change

Adds a `TypeAhead` filter to filter projects by country of origin. Refactored to use a single `RoutedTypeahead` component instead of a `FilterSectorTypeahead` as well as a `FilterCountryTypeahead`.

## Test instructions

On `/investments/projects` - a new filter for country of origin should appear. Typing in the input should show suggestions allowing multiple countries to be picked. These countries should then appear as chips above the list of projects. It should be possible to clear the chips by clicking on them or the clear all button.

## Screenshots
Screenshot from 2020-12-02 16-44-07

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/apps/investments/client/projects/metadata/countries.js [hunk 1]
```diff
@@ -0,0 +1,876 @@
+export const countryOptions = [
```
Comment (tomkingston) [inline src/apps/investments/client/projects/metadata/countries.js:1]: I took these values from a fixture in the api - please let me know if they look correct!

File: src/apps/investments/client/projects/metadata/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+export { countryOptions } from './countries'
+export { sectorOptions } from './sectors'
```
Comment (tomkingston) [inline src/apps/investments/client/projects/metadata/index.js:2]: Moved these out to their own files to make the code a bit easier to navigate

File: src/client/components/Filters/elements/FilterAdvisersTypeahead/index.jsx [hunk 3]
```diff
@@ -37,7 +37,7 @@ const StyledFieldWrapper = styled(FieldWrapper)`
   }
 `
 
-const FilterAdvisersTypeAhead = ({
+const FilterAdvisersTypeahead = ({
```
Comment (tomkingston) [inline src/client/components/Filters/elements/FilterAdvisersTypeahead/index.jsx:40]: Just updated the casing on this to be consistent with the `Typeahead` component

File: src/apps/investments/client/projects/metadata/countries.js (hunk 1 above)
Comment (rachaelcodes) [reply src/apps/investments/client/projects/metadata/countries.js:1]: Looks ok to me - the values match the filter queries in the dev and live events. Plus, Hong Kong's included, which means we've probably got the right country list.

**Review by rachaelcodes — APPROVED** [review]
Looks good 👍

**Review by ambrozic — APPROVED** [review]

**Pull request merged**